Android/app/src/main/java/com/example/boardgamesocial/HomePostActivity.java

```diff
@@ -55,4 +58,14 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
                 return super.onOptionsItemSelected(item);
         }
     }
+
+    public static Intent getIntent(Context context) {
+        System.out.println("Inside HomePostActivity getIntent!");
+        return new Intent(context, HomePostActivity.class);
+    }
+
+    private void switchToGamesCollectionActivity() {
+        Intent goToGamesCollectionActivity = GamesCollectionActivity.getIntent(HomePostActivity.this);
+        startActivity(goToGamesCollectionActivity);
+    }
```

Comment:
I thought we agreed on using fragments, right?

Reply:
Yes, this is a temporary change added. Once the fragment changes we did on Tuesday are pushed on git, I would be adding those.